renderer/index.js

```diff
@@ -213,6 +211,16 @@ function setupIpc () {
   })
 }
 
+function detectDevices () {
+  chromecasts.on('update', function (player) {
+    state.devices.chromecast = player
+  })
+
+  airplay.createBrowser().on('deviceOn', function (player) {
+    state.devices.airplay = player
+  }).start()
+}
```

Comment:
also moved device detection into its own function and removed the `update()` inside the chromecasts callback. I figure we're generally covered on updates.